__tests__/app.test.js

```diff
@@ -55,395 +55,425 @@ describe('getTopics', () => {
 //Users tests
 
 describe('getUsers', () => {
-
   test('200: /api/users responds with an array', () => {
     return request(app)
-    .get('/api/users')
-    .expect(200)
-    .then((res) => {
-      expect(res.body).toBeInstanceOf(Object);
-      expect(res.body.users).toBeInstanceOf(Array)
-    });
+      .get('/api/users')
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toBeInstanceOf(Object);
+        expect(res.body.users).toBeInstanceOf(Array);
+      });
   });
 
   test('200 /api/topics responds with an array of users, formatted correctly', () => {
     return request(app)
-    .get('/api/users')
-    .expect(200)
-    .then((res) => {
-      expect(
-        res.body.users.forEach((user) => {
-          expect(user).toMatchObject({
-            username: expect.any(String),
-          });
-        })
+      .get('/api/users')
+      .expect(200)
+      .then((res) => {
+        expect(
+          res.body.users.forEach((user) => {
+            expect(user).toMatchObject({
+              username: expect.any(String),
+            });
+          })
         );
       });
-    });
+  });
 
   test('/api/userr responds 404 not found', () => {
-      return request(app)
+    return request(app)
       .get('/api/userr')
       .expect(404)
       .then((result) => {
         expect(result.body.msg).toBe('invalid path');
       });
-    });
   });
+});
+
+describe('getUserById', () => {
+  test('200: /api/users/:username responds with an object with username, avatar_url, name', () => {
+    return request(app)
+      .get('/api/users/rogersop')
+      .expect(200)
+      .then((result) => {
+        expect(result.body.user).toEqual({
+          username: 'rogersop',
+          name: 'paul',
+          avatar_url:
+            'https://avatars2.githubusercontent.com/u/24394918?s=400&v=4',
+        });
+      });
+  });
+  test('/api/user/i_dont_exist responds 404 not found', () => {
+    return request(app)
+      .get('/api/i_dont_exist')
```

Comment:
Your test say `api/user/i_dont_exist` but your get request is `api/i_dont_exist`